.github/workflows/ci-windows-artifacts.yml

```diff
@@ -38,26 +38,25 @@ jobs:
       if: steps.cache-ext.outputs.cache-hit != 'true'
       working-directory: ./ext
       run: ./aom.cmd
+      # Visual Studio 2022 has an issue starting at 17.8.0 which might cause
+      # AVX-512 instructions to be emitted in non-AVX-512 code. See
+      # https://github.com/AOMediaCodec/libavif/issues/2033#issuecomment-1960062751.
+      env:
+        CC: clang-cl
+        CXX: clang-cl
     - name: Build dav1d
       if: steps.cache-ext.outputs.cache-hit != 'true'
       working-directory: ./ext
       run: ./dav1d.cmd
     - name: Build libyuv
       if: steps.cache-ext.outputs.cache-hit != 'true'
       working-directory: ./ext
+      run: ./libyuv.cmd
       # Use clang-cl to build libyuv. The assembly code in libyuv is written in the
       # GCC inline assembly format. Visual C++ doesn't support GCC inline assembly.
-      # This is PowerShell syntax.
-      run: >
-        $Env:CC = 'clang-cl'
-
-        $Env:CXX = 'clang-cl'
-
-        ./libyuv.cmd
-
-        $Env:CC = ''
-
-        $Env:CXX = ''
+      env:
+        CC: clang-cl
+        CXX: clang-cl
```

Comment:
Yannis: I found the `env` key, documented at https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow.